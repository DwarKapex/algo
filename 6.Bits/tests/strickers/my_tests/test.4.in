8/8/5p2/8/8/2B5/3P4/8
