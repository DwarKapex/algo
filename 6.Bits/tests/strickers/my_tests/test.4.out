0
35257554567681
0